### PR TITLE
webpack-based build

### PR DESCRIPTION
**Pull request by xkxd**
pros
* es6 + modules
* live reload
* no dev.css in repo
* no jsons concatenation
* source maps
* single index.html

cons
* gzipped size went up from 6,5KB to 7 KB  
* \<add additional ones\>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,13 +18,20 @@
...
   "devDependencies": {
-    "autoprefixer": "7.1.6",
-    "design-system": "git+https://github.com/Wikia/design-system.git#8.11.1",
-    "http-server": "0.10.0",
-    "node-sass": "4.5.3",
-    "nodemon": "1.12.1",
-    "npm-run-all": "4.1.1",
-    "postcss-cli": "4.1.1",
-    "uglify-js": "3.1.5"
+    "autoprefixer": "^7.1.6",
```
Comment (latata) [inline package.json]: I'd keep versions without `^`

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+    ["env", {
+      "useBuiltIns": true,
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7", "ie 10"]
```
Comment (bkoval) [inline .babelrc]: IE11+

File: package.json [hunk 3]
```diff
@@ -1,22 +1,11 @@
 {
   "name": "jwplayer-fandom",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (bkoval) [inline package.json]: Are we sure about it? ;>

File: postcss.config.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	plugins: [
+		require('autoprefixer')({})
```
Comment (bkoval) [inline postcss.config.js]: Would that somehow share babel config for supported browsers? If not, we should probably add it at some point :)

**Pull request closed without merging**